scheduler/pkg/store/pipeline/store.go

```diff
@@ -293,7 +291,8 @@ func (ps *PipelineStore) GetAllRunningPipelineVersions() []coordinator.PipelineE
 	for _, p := range ps.pipelines {
 		pv := p.GetLatestPipelineVersion()
 		switch pv.State.Status {
-		case PipelineCreate, PipelineCreating, PipelineReady:
+		// we consider PipelineTerminating as running as it is still active
+		case PipelineCreate, PipelineCreating, PipelineReady, PipelineTerminating:
```

Comment:
Would it be useful to document the meaning of the Terminating states?
The way I see it now:
- Terminate: Received signal that pipeline should be terminated
- Terminating: In the process of doing cleanup/housekeeping for pipeline termination.
- Terminated: All finished

Would that be an accurate description? How long do you think the "Terminating" state would last?

Reply:
done in [ac56141](https://github.com/SeldonIO/seldon-core/pull/5298/commits/ac5614142e8cf43948478a75dbe02fe847567acd)